docs/fsharp/language-reference/reference-cells.md

```diff
@@ -101,6 +98,45 @@ The following code examples demonstrate the use of reference cells in closures.
 
 [!code-fsharp[Main](../../../samples/snippets/fsharp/lang-ref-1/snippet2207.fs)]
 
+## Consuming C#7 and VB15 `ref` returns
```

Comment:
I suggest we avoid all mention of Visual Basic in the docs.  There's just no point. Use either "C#" or "Consuming ref returns from other .NET languages"